main.js

```diff
@@ -0,0 +1,134 @@
+let encryptedWord = [];
+let startPage = document.getElementById("word-input");
+let gallowsPage = document.getElementById("word-guessing");
+let wordToGuess;
+let mismatchedLetters = 0;
+
+// at each point in the game, the second player can reset the game
+restart.onclick = () => {
+    location.reload(); 
+}
+
+// determines if the word is complete or still encrypted
+// depending on the situation, one image is displayed
+function allLettersMatch(encryptedWord) {
+    let allLetters = 1;
+    for (let i = 0; i <= encryptedWord.length; ++i) {
+        if (!isLetter(encryptedWord[i])) {
+            allLetters = 0;
+        }
+    }
+    if (allLetters == 1) {
+        mismatchedLetters = 8;
+        hangmanStatus(mismatchedLetters);
+    }
+}
+
+// if the id of the clicked button matches one letter of the word
+// the letter will be displayed
+// we return the truth of the condition in order to change/not change the image
+function completeWord(id) {
+    let letterMatch = 0;
+    for (let i = 0; i < wordToGuess.length; ++i) {
+        if (wordToGuess[i].toLowerCase() == id.toLowerCase()) {
+            encryptedWord[i] = id;
+            letterMatch = 1;
+        };
+    }
+    return letterMatch;
+}
+
+// if all the letters in the word are matched, the game will be over
+// if all the lifes are wasted, the game is also over
+function countLifes(letterMatch) {
+    if (letterMatch == 0) {
+        ++mismatchedLetters;
+        if (mismatchedLetters == 8) {
+            mismatchedLetters = 9;
+            hangmanStatus(mismatchedLetters);
+        }
+    }
+    allLettersMatch(encryptedWord);
+}
+
+// displayes the letters of the alphabet as buttons
+// adds functionality to the buttons
+function displayAlphabet() {
+    for (let i = 0; i < alphabet.length; ++i) {
+        const currentButton = document.createElement("button");
+        letters.appendChild(currentButton);
+        currentButton.innerHTML = alphabet[i];
+        currentButton.id = alphabet[i];
+        currentButton.addEventListener("click", ()=> playerGuess(currentButton.id));
+        currentButton.className = "btn btn-light";
+    }
+}
+
+// for each wrong letter, the man is closer to his death
+function displayImages(mismatchedLetters) {
+    images.src = gallowImages[mismatchedLetters];
+}
+
+// for each letter in the word, a " _ " will be displayed
+function encryptLetters(wordToGuess) {
+    for (let i = 0; i < wordToGuess.length; ++i) {
+        encryptedWord.push(" _ ");
+    }
+}
+
+// displaying only the reset button and the last image if the game is over
+function hangmanStatus(mismatchedLetters) {
+    displayImages(mismatchedLetters);
+    encrypted.style.display = "none";
+    letters.style.display = "none";
+}
+
+// determine if the current character is a letter;
+function isLetter(char) {
```

Comment:
Can you get rid of this function? And no, I don't mean to move the code from this function where you need it, like, completely remove the code.